File: 2021 maj/python/3.py
with open("instrukcje.txt", "r") as f:
    inst = f.read().strip().split('\n')

d = dict()

for line in inst:
    s, c = line.split()
    
    if s == "DOPISZ":
        if c not in d.keys():
            d[c] = 0
        d[c] += 1

best = max(d.values())
for i, j in d.items():
    if j == best:
        print(i, j)

File: 2021 maj/c++/3.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int cnt[333];

int main()
{
    ifstream in("instrukcje.txt");
    
    string s;
    char c;
    while (in >> s >> c)
    {
        if (s == "DOPISZ")
        {
            cnt[c]++;
        }
    }
    char ans = 'A';
    for (char c = 'B'; c <= 'Z'; c++)
        if (cnt[c] > cnt[ans])
            ans = c;
    cout << ans << ' ' << cnt[ans] << '\n';
}